YAKL_init.h

```diff
@@ -96,3 +99,129 @@
 
     yakl_mtx.unlock();
   } //
+
```

Comment:
Was this part a mistake? It looks like `YAKL_alloc_free.h` was manually included here.

Reply:
The header was located same as in the master: https://github.com/abagusetty/YAKL/blob/add-sycl/YAKL_init.h#L4